### DISCUSSION TITLE
npm install from GitHub

### DISCUSSION THREAD

**Discussion by pbougie** [tinacms/tinacms#3866] [Help] [answered]
Is there a way to install TinaCMS directly from the `main` branch on GitHub? It's not straightforward because it's a monorepo.  There are a few critical fixes that the team has done but have not been released to `npm`. Without these updates, my clients are in limbo since Forestry was discontinued. Unfortunately, there are sections in their CMS that can't be updated until these needed updates are available.

**Comment by logan-anderson** [comment, marked as answer]
@pbougie We are planning on doing a release soon (Planning for today). You can install the latest packages (From main on github) by using the `@beta` tag. 

EX:
`yarn add tinacms@beta @tinacms/cli@beta` however this may cause issues in production since Tina Cloud does not get the latest version until it is released.

**Comment by pbougie** [comment]
@logan-anderson Understood about the `@beta` tag and Tina Cloud. I can wait for the release if it's later today. Thanks for replying.

**Reply by logan-anderson** [reply]
The release is out and you should be able to update now with no issues.

**Comment by pbougie** [comment]
@logan-anderson Thanks.
